src/main/kotlin/com/pubnub/api/PubNub.kt

```diff
@@ -2283,15 +2313,32 @@ class PubNub internal constructor(
         return UuidMetadataImpl(this, ChannelName(id))
     }
 
-    fun channelGroup(name: String): ChannelGroup {
-        return ChannelGroupImpl(this, ChannelGroupName(name))
-    }
-
+    /**
+     * Create a [SubscriptionSet] from the given [subscriptions].
+     *
+     * @param subscriptions the subscriptions that will be added to the returned [SubscriptionSet]
+     * @return a [SubscriptionSet] containing all [subscriptions]
+     */
     fun subscriptionSetOf(subscriptions: Set<Subscription> = emptySet()): SubscriptionSet {
         @Suppress("UNCHECKED_CAST")
         return SubscriptionSetImpl(this, subscriptions as Set<SubscriptionImpl>)
     }
 
+    /**
+     * Create a [SubscriptionSet] containing [Subscription] objects for the given sets of [channels] and
+     * [channelGroups].
+     *
+     * Please note that the subscriptions are not active until you call [SubscriptionSet.subscribe].
+     *
+     * This is the same as calling [PubNub.channel] followed by [Channel.subscription] for each channel,
+     * then creating a [subscriptionSetOf] using these [Subscription] objects (similarly for channel groups).
+     *
+     * @param channels the channels to create subscriptions for
+     * @param channelGroups the channel groups to create subscriptions for
+     * @param options the [SubscriptionOptions] to pass for each subscription. Refer to supported options in [Channel] and
+     * [ChannelGroup] documentation.
+     * @return a [SubscriptionSet] containing subscriptions for the given [channels] and [channelGroups]
+     */
     fun subscriptionSetOf(
```

Comment:
It would be valuable to have integration test that demonstrate usage of "subscriptionSetOf"